src/Jenssegers/Mongodb/Connection.php

```diff
@@ -65,6 +68,17 @@ public function __construct(array $config)
         $this->useDefaultQueryGrammar();
     }
 
+    /**
+     * Dynamically pass methods to the connection.
+     * @param string $method
+     * @param array $parameters
+     * @return mixed
+     */
+    public function __call($method, $parameters)
```

Comment:
Is this method necessary for transactions? If so, can you explain?

Reply:
I just move this function to top after __construct method